packages/cli/cli/src/commands/generate/generateAPIWorkspaces.ts

```diff
@@ -52,6 +54,23 @@ export async function generateAPIWorkspaces({
         token = currentToken;
     }
 
+    if (!isCI()) {
```

Comment:
nit: thoughts on adding a `--force` option so that users never get prompted? 

(not blocking merge at all)

Reply:
I think this would be great to have, will add tomo